### PR TITLE
fix: skip broken builds for now

### PR DESCRIPTION
**Pull request by henryiii**
First step from #112. Follow up PRs can try enabling these again.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,6 +13,9 @@ build:
   number: 0
   script:
     - {{ PYTHON }} -m pip install . -vvv
+  skip: true  # [osx and build_platform != target_platform]
+  skip: true  # [py37]
```
Comment (henryiii) [inline recipe/meta.yaml:17]:
```suggestion
  skip: true  # [py==37]
```

**Comment by henryiii** [comment]
@conda-forge-admin, please rerender

**Comment by maxbachmann** [comment]
I am not familar with the syntax for the exclusion. It appears we do now skip:
- OSX arm64
- pypy
- python3.7

It would be great to restrict this further to:
- OSX arm64
- (pypy and Windows) or (pypy and Linux  and (arm64 | ppc64le)
- python3.7 and Linux  and (arm64 | ppc64le)

**Comment by maxbachmann** [comment]
@conda-forge-admin, please rerender

**Comment by henryiii** [comment]
I copied what was in the issue, though I was wondering about why all 3.7 was being excluded.

**Comment by maxbachmann** [comment]
I added the skip in jarowinkler as well: https://github.com/conda-forge/jarowinkler-feedstock/pull/16

**Comment by maxbachmann** [comment]
> Not all 3.7 are problematic, but might as well skip because we will likely drop 3.7 soonish anyway...

@ngam is there any ETA for when Python 3.7 will be dropped?

**Pull request merged**